corehq/apps/userreports/models.py

```diff
@@ -525,6 +526,10 @@ def _all(cls):
     @classmethod
     def all(cls):
         for wrapped, path in cls._all():
+            if (wrapped.server_environment and
```

Comment:
Eventually this should be required, but just want to get the icds reports off prod ES